app/src/lib/editors/shared.ts

```diff
@@ -0,0 +1,76 @@
+import * as Fs from 'fs'
+
+type ProgramNotFound = {
+  name: string
```

Comment:
Actually, I think I just got grumpy at how interfaces all need to start with `I` and switched back - so for cosmetic reasons basically :trollface: